### PR TITLE
	new file:   sway_nl.md

### PR DESCRIPTION
**Pull request by SleepyMario**
This is the Dutch translation of the readme. I will take care of the rest of the docs as well.

### CODE REVIEW

File: sway_nl.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Sway
+
+English - 日本語 - Deutsch - Ελληνικά - Français - Español - Українська - Italiano - Nederlands
+
+“**S**irCompwn’s **Way**land compositor” is een **werk-in-uitvoering** zijnde i3-compatible [Wayland](http://wayland.freedesktop.org/) compositor. Lees Voor meer informatie a.u.b. the [FAQ](https://github.com/SirCmpwn/sway/wiki) of bezoek het IRC kanaal (#sway op irc.freenode.net).
```
Comment (SirCmpwn) [inline sway_nl.md]: Do not localize my name

**Comment by SirCmpwn** [comment]
You need to follow the [i18n guidelines](https://github.com/SirCmpwn/sway/issues/1318)

File: sway_nl.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+# Starten
+
+Voer ```sway``` uit vanuit een TTY. Sommige Display Managers zouden kunnen werken maar worden niet direct ondersteund door Sway (van gdm is het bekend dat het redelijk goed werkt.)
+
+# Ondersteuning in het Nederlands
+
+SleepyMario biedt ondersteuning in het Nederlands op IRC en GitHub, maar op onregelmatige tijdstippen. Laat aub. een berichtje achter en hij zal u z.s.m te woord staan.  
+
+Stuur een email naar [SleepyMario](theonesleepymario@gmail.com) voor suggesties wat betreft deze vertaling.
```
Comment (SirCmpwn) [inline sway_nl.md]: Translation improvements are to be done via GitHub pull requests.

**Comment by alexbakker** [comment]
The translation looks alright. No obvious mistakes.

File: sway_nl.md [hunk 3]
```diff
@@ -81,5 +79,5 @@ Voer ```sway``` uit vanuit een TTY. Sommige Display Managers zouden kunnen werke
 
 SleepyMario biedt ondersteuning in het Nederlands op IRC en GitHub, maar op onregelmatige tijdstippen. Laat aub. een berichtje achter en hij zal u z.s.m te woord staan.  
 
-Stuur een email naar [SleepyMario](theonesleepymario@gmail.com) voor suggesties wat betreft deze vertaling. 
+Suggesties voor een betere vertaling graag via [GitHub](https://github.com/sleepymario).
```
Comment (SirCmpwn) [inline sway_nl.md]: GitHub pull requests against Sway, like this one. They shouldn't reach out to you directly, I will loop you into the review here on GitHub.

Comment (SirCmpwn) [inline sway_nl.md]: You also need to move the section on Dutch info up, *please* review the TL guidelines.

**Comment by SleepyMario** [comment]
Yeah it's a bit archaic, which i intend to change. I'm still looking at the right mix between casual and formal language. I don't want to resort to baby-language but turning doing it too formal is odd as well.

**Comment by SleepyMario** [comment]
@alexbakker Suggestions are appreciated.

**Comment by SleepyMario** [comment]
Come on @alexbakker , if you're that quick to comment on translation quality, you _should_ be capable of producing _something_.

And if you were offline then I'm sure you were online when you made your intelligent comments. Now, please, can I have some constructive input please?

**Comment by SirCmpwn** [comment]
Chill the fuck out, @SleepyMario. We aren't in a hurry here and you need to stop badgering him.

**Comment by SirCmpwn** [comment]
Maybe we'll get a Dutch translation from someone who is less of an asshole.

**Pull request closed without merging**